templates/ts-apollo-mongodb-backend/README.md

```diff
@@ -52,14 +30,59 @@ yarn develop
 
 For more on customising your Graphback application, check out [our docs](https://graphback.dev/docs/gettingstarted)
 
-If your project contains client application please follow `./client/README.md` for running client side application
+If your project contains the client application then please follow `./client/README.md` for info regarding running the client side.
+
+### Dependencies and Tools
 
-## Re-generating types from schema
+The following tools and technologies have been used to create this template:
 
-If you made made changes to your business model, it's advised to regenerate the `generated-types.ts` which contains Typescript file of your business entities. 
+- [GraphQL](https://graphql.org/): GraphQL is an open-source data query and manipulation language for APIs which was publically released by Facebook in 2015.
+
+- [Apollo Server](https://www.apollographql.com/docs/apollo-server/): Apollo Server provides a way to simplify buildiing the GraphQL server. It can be used with several popular libraries for Node.js like Express, Koa, Hapi. Here we have used it with Express.
```

Comment:
```suggestion
- [Apollo Server](https://www.apollographql.com/docs/apollo-server/): Apollo Server provides a way to simplify building the GraphQL server. It can be used with several popular libraries for Node.js like Express, Koa, Hapi. Here we have used it with Express.
```